### DISCUSSION TITLE
What graphics design guidelines that Material for Mkdocs follow?

### DISCUSSION THREAD

**Discussion by ooker777** [squidfunk/mkdocs-material#5469] [General]
It seems obvious, but I'm unable to find any info in the docs. Just to make sure, is it [Material Design](https://m3.material.io/)? If so, why not saying it anywhere?

**Comment by squidfunk** [comment]
It followed the Material Design v1 spec closely, but not the newer specs (v2 and v3). This is also why it is not linked, because some users asked about it, and honestly, I find the newer specs to be worse iterations of v1. However, we say so in the README.

**Comment by ooker777** [comment]
Should we add that to the doc?

**Reply by squidfunk** [reply]
No. As said, the README mentions it.

**Reply by ooker777** [reply]
sorry I only read via the email, so I didn't see your edit. But well, the link links to v3. Perhaps we want to explicitly link to the v1
